### PR TITLE
chore: restructure utility components

### PR DESCRIPTION
**Pull request by richbachman**
- [x] move absolute, aspect-ratio, flex, grid, media-object to layout
- [x] move hyphenate, screen-reader, truncate to components
- [x] move unstyled-list to primitives
- [x] delete long-form package
- [x] update storybook stories
- [x] fixed 404 pages for primitives and layout

### CODE REVIEW

File: packages/paste-website/src/components/empty-state/Error404.tsx [hunk 1]
```diff
@@ -25,20 +26,17 @@ const Error404 = ({pathname, componentList, utilitityList}: Error404Props): Reac
...
+  /*
   // TODO populate graphql first
   if (pathname.includes(SidebarCategoryRoutes.PRIMITIVES) && primitiveList.includes(packageName)) {
       <NotBuilt type="primitive" name={name} />;
   }
+
+  if (pathname.includes(SidebarCategoryRoutes.LAYOUT) && layoutList.includes(packageName)) {
+      <NotBuilt type="layout" name={name} />;
+  }
   */
```
Comment (richbachman) [inline packages/paste-website/src/components/empty-state/Error404.tsx]: Added `layoutList` to this todo as well. Spreading `layoutList` or `primitiveList` above causes an error in the custom 404 page.

image

Comment (richbachman) [reply packages/paste-website/src/components/empty-state/Error404.tsx]: This should be fixed now.

**Comment by SiTaggart** [comment]
Don’t forget to bust that CircleCi Cache to get Applitools working

**Comment by richbachman** [comment]
@SiTaggart Will do. Was just waiting to bust the cache until everyone reviewed in case there are other structure changes.

File: packages/paste-core/components/aspect-ratio/stories/index.stories.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import {Box} from '@twilio-paste/box';
 import {Absolute} from '@twilio-paste/absolute';
 import {AspectRatio} from '../src';
 
-storiesOf('Utilities|Aspect Ratio', module)
+storiesOf('Components|Aspect Ratio', module)
```
[the remark this answers is not in the document]
Comment (richbachman) [reply packages/paste-core/components/aspect-ratio/stories/index.stories.tsx]: Moving this to a layout component.

**Pull request merged**